tests/helpers/start-app.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
+import './simple-auth';
```

Comment:
Without it, the tests fail with `ReferenceError: authenticateSession is not defined`